_includes/android/objects.md

```diff
@@ -64,12 +64,13 @@ boolean cheatMode = gameScore.getBoolean("cheatMode");
 
 If you don't know what type of data you're getting out, you can call `get(key)`, but then you probably have to cast it right away anyways. In most situations you should use the typed accessors like `getString`.
 
-The three special values have their own accessors:
+The four special values have their own accessors:
 
 ```java
 String objectId = gameScore.getObjectId();
 Date updatedAt = gameScore.getUpdatedAt();
 Date createdAt = gameScore.getCreatedAt();
+ParseACL acl = gameScore.getACL();
```

Comment:
This is a good point. Do you mind doing it for the other docs as well? The syntax shouldn't be too different and you can just base it off surrounding code if you're not familiar with the language.